src/rdkafka.h

```diff
@@ -7337,8 +7629,209 @@ RD_EXPORT const rd_kafka_topic_partition_list_t *
+/**@}*/
+
+/**
+ * @name Admin API - DescribeGroups
+ * @{
+ */
+
+/**
+ * @brief DescribeGroups result type.
+ *
+ */
+typedef struct rd_kafka_ConsumerGroupDescription_s
+    rd_kafka_ConsumerGroupDescription_t;
+
+/**
+ * @brief Member description included in ConsumerGroupDescription.
+ *
+ */
+typedef struct rd_kafka_MemberDescription_s rd_kafka_MemberDescription_t;
+
+/**
+ * @brief Member assignment included in MemberDescription.
+ *
+ */
+typedef struct rd_kafka_MemberAssignment_s rd_kafka_MemberAssignment_t;
+
+/**
+ * @brief Describe groups from cluster as specified by the \p groups
+ *        array of size \p groups_cnt elements.
+ *
+ * @param rk Client instance.
+ * @param groups Array of groups to describe.
+ * @param groups_cnt Number of elements in \p groups array.
+ * @param options Optional admin options, or NULL for defaults.
+ * @param rkqu Queue to emit result on.
+ *
+ * @remark The result event type emitted on the supplied queue is of type
+ *         \c RD_KAFKA_EVENT_DESCRIBEGROUPS_RESULT
+ */
+RD_EXPORT
+void rd_kafka_DescribeGroups(rd_kafka_t *rk,
+                             const char **groups,
+                             size_t groups_cnt,
+                             const rd_kafka_AdminOptions_t *options,
+                             rd_kafka_queue_t *rkqu);
+
+/**
+ * @brief Get an array of group results from a DescribeGroups result.
+ *
+ * The returned groups life-time is the same as the \p result object.
+ *
+ * @param result Result to get group results from.
+ * @param cntp is updated to the number of elements in the array.
+ */
+RD_EXPORT
+const rd_kafka_ConsumerGroupDescription_t **
+rd_kafka_DescribeGroups_result_groups(
+    const rd_kafka_DescribeGroups_result_t *result,
+    size_t *cntp);
+
+
+/**
+ * @brief Gets the group id for the \p desc group.
+ *
+ * @param desc The group description.
+ * @return The group id, or NULL if \p desc is NULL.
+ */
+RD_EXPORT
+const char *rd_kafka_ConsumerGroupDescription_group_id(
+    const rd_kafka_ConsumerGroupDescription_t *desc);
+
+/**
+ * @brief Gets the error for the \p desc group.
+ *
+ * @param desc The group description.
+ * @return The group description error, or NULL if no error or \p desc is NULL.
+ */
+RD_EXPORT
+const rd_kafka_error_t *rd_kafka_ConsumerGroupDescription_error(
+    const rd_kafka_ConsumerGroupDescription_t *desc);
+
+/**
+ * @brief Is the \p desc group a simple consumer group.
+ *
+ * @param desc The group description.
+ * @return 1 if the group is a simple consumer group,
+ *         else 0 (also if \p desc is NULL).
+ */
+RD_EXPORT
+int rd_kafka_ConsumerGroupDescription_is_simple_consumer_group(
+    const rd_kafka_ConsumerGroupDescription_t *desc);
+
+
+/**
+ * @brief Gets the partition assignor for the \p desc group.
+ *
+ * @param desc The group description.
+ * @return The partition assignor, or NULL if \p desc is NULL.
+ */
+RD_EXPORT
+char *rd_kafka_ConsumerGroupDescription_partition_assignor(
```

Comment:
const char *

Also add:
`@remark The lifetime of the returned string is the same as the \p grpdesc object.`

Reply:
done